EIPS/eip-42.md

```diff
@@ -0,0 +1,399 @@
+---
+eip: 42 (final number to be confirmed)
+title: T-REX - Token for Regulated EXchanges
+authors: Joachim Lebrun, Tony Malghem, Kevin Thizy, Luc Falempin, Adam Boudjemaa
+type: Standards Track
+category: ERC
+status: Draft
+require: ERC-20 , ERC-734, ERC-735, ERC-1822
+created: 2021-07-09
+---
+
+### Note : we use the EIP n°42 in this proposition document but the final number needs to be determined by an EIP Editor, nonetheless the n°42 will be used until we have a final EIP number to refer to.
+
+## Simple Summary
+
+The T-REX (ERC-42) token is an institutional grade security token standard, developed on Ethereum. <br>
+This standard provides a library of interfaces for the management and transfer of security tokens. It allows the contract owner/agent to manage the rules that the transfers have to fulfill to be considered as valid transfers as well as the rules related to an onchain identity to consider its owner as an eligible participant in the token ecosystem. The standard also provides a full set of functions to allow the token agent to manage the token properly.
```

Comment:
The simple summary should only be 1-2 sentences, please consolidate to that length.